Src/DependencyCollector/Shared/Implementation/ProfilerSqlConnectionProcessing.cs

```diff
@@ -0,0 +1,72 @@
+﻿namespace Microsoft.ApplicationInsights.DependencyCollector.Implementation
+{
+    using System;
+    using System.Data.SqlClient;
+    using System.Threading.Tasks;
```

Comment:
For casting to Task

---
In reply to: [97904986](https://github.com/Microsoft/ApplicationInsights-dotnet-server/pull/298#discussion_r97904986) [](ancestors = 97904986)